src/fidesctl/ctl/core/config/__init__.py

```diff
@@ -49,6 +51,30 @@ def handle_deprecated_fields(settings: MutableMapping) -> MutableMapping:
     return settings
 
 
+def handle_deprecated_env_variables(settings: MutableMapping) -> MutableMapping:
+    """
+    Custom logic for handling deprecated ENV variable configuration.
+    """
+
+    deprecated_env_vars = regex(r"FIDESCTL__API__(\w+)")
```

Comment:
@ThomasLaPiana I believe this should cover all of the deprecated ENV variables? We can always add more like:
```py
deprecated_env_vars = regex(r"FIDESCTL__[API|FOO|BAR]__(\w+)")
```

Reply:
This covers all of them, thanks! 👍